data_validation/combiner.py

```diff
@@ -392,16 +394,30 @@ def _join_pivots(source, target, differences, join_on_fields):
     return joined
 
 
-def _add_metadata(joined, run_metadata):
+def _add_metadata(joined: types.relations.Table, run_metadata: metadata.RunMetadata, source_df: DataFrame, target_df : DataFrame, differences_df: DataFrame):
     # TODO: Add source and target queries to metadata
     run_metadata.end_time = datetime.datetime.now(datetime.timezone.utc)
 
+    # maybe check how to restrict logic below for row validations only?
     joined = joined[
         joined,
         ibis.literal(run_metadata.run_id).name("run_id"),
         ibis.literal(run_metadata.labels).name("labels"),
         ibis.literal(run_metadata.start_time).name("start_time"),
         ibis.literal(run_metadata.end_time).name("end_time"),
+        # problem: current always prints table names for the amount of rows, but it saves on BQ, what is already something
+        ibis.literal(str({
```

Comment:
Need to consider JSON and CSV output too.

I'm not a fan of adding this metadata to the result data frame and think we should discuss other options.